Gemfile

```diff
@@ -10,7 +10,8 @@ group :development, :test do
   gem 'bundler'
   gem 'hashie'
   gem 'rake'
-  gem 'rubocop', '0.51.0'
+  gem 'rubocop'
```

Comment:
We don't commit Gemfile.lock, so you should specify a version here or future builds will break for no reason as newer versions get released.